build.sbt

```diff
@@ -5,14 +5,21 @@ val appName = "irr-api-end-to-end-service-guide"
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
```

Comment:
Can SbtAutoBuildPlugin(See [here)](https://github.com/hmrc/sbt-auto-build#upgrading-to-the-latest-release)) since it can be removed for version 3.0.0 upwards. Also, can this be removed SbtGitVersioning and SbtArtifactory?

Reply:
you meant to remove only SbtGitVersioning and SbtArtifactory but not SbtAutoBuildPlugin(as it contains these 2)

Reply:
We can also remove SbtAutoBuildPlugin from here, since the version is > than 3.0.0 and it is auto-enabled and we want it enabled. See [here](https://github.com/hmrc/sbt-auto-build#upgrading-to-the-latest-release).